server/api/teams/resources.py

```diff
@@ -293,8 +319,37 @@ def get(self):
             500:
                 description: Internal Server Error
         """
+
+        filters = {}
+        filters["team_name_filter"] = request.args.get("team_name")
```

Comment:
Would it be possible to filter by the presence of the string instead of an exact match?